modules/dtg/manifests/backup.pp

```diff
@@ -76,20 +76,27 @@
 # The name of the backup will be used as the directory name for the subdir containing the backups
 # user is the user to ssh in as
 # host is the host to ssh into
-define dtg::backup::hostsetup($user, $host, $weekday) {
+define dtg::backup::hostsetup($user, $group, $host, $weekday) { #needs to have a configurable backup group + default
   $backupsdirectory = $dtg::backup::host::directory
   $backupsuser      = $dtg::backup::host::user
   $backupskey       = $dtg::backup::host::realkey
   $backupto = "${backupsdirectory}/${name}"
+
+  # Default to the group and user being the same
+  if $group == undef {
+    $group = $dtg::backup::host::user
+  }
+
   file {"${backupto}":
     ensure => directory,
     owner  => $backupsuser,
-    group  => $backupsuser,
+    group  => $group,
     mode   => '0700',
   }
   cron {"backup ${name}":
     ensure  => present,
     user    => $backupsuser,
+    environmental => "MAILTO=dtg-infra@cl.cam.ac.uk",
```

Comment:
"environment" not "environmental" according to https://docs.puppetlabs.com/references/latest/type.html#cron